.github/labeler.yml

```diff
@@ -0,0 +1,30 @@
+documentation:
+  - changed-files:
+    - any-glob-to-any-file:
+      - 'README*'
+      - 'CONTRIBUTING*'
+      - 'code-of-conduct*'
```

Comment:
`doc` would be shorter and better?

Reply:
[7a5d7ee](https://github.com/percona/percona-server-mongodb-operator/commit/7a5d7ee2b2631682ee73f4c6824eb15e73397204)